.travis.yml

```diff
@@ -14,8 +14,8 @@ script:
 - fastlane test
 
 after_failure:
-- if [ $TRAVIS_PULL_REQUEST_SLUG == "aol-public/OneMobileSDK-controls-ios" ]; then . manage-diffs.sh; fi
+- if [[ $TRAVIS_PULL_REQUEST_SLUG == "aol-public/OneMobileSDK-controls-ios" ]]; then . manage-diffs.sh; fi
 
 
 after_success:
-- if [ "$TRAVIS_PULL_REQUEST_SLUG" == "aol-public/OneMobileSDK-controls-ios" ]; then . manage-diffs.sh; fi
+- if [[ $TRAVIS_PULL_REQUEST_SLUG == "aol-public/OneMobileSDK-controls-ios" ]]; then . manage-diffs.sh; fi
```

Comment:
`after_success` have never been working?

Reply:
It didn't, but not because of that. I've just removed a warning in this line. But the reason that it wasn't working is that after migration on another Travis (.org -> .com) that repo had no env variables with aws and danger tokens.